### DISCUSSION TITLE
On the case of using list type data inside a map

### DISCUSSION THREAD

**Discussion by james5haumor** [jorgen/json_struct#54] [General]
I have a situation like this:

```cpp
struct mystruct
{
    std::map<std::string,std::list<std::string>> dataMap;
    JS_OBJ(dataMap);
};
```

But when compiling the code, it throws an error. Is it currently not supported to handle this type of structure? 😭

**Comment by jorgen** [comment]
Hi! Thank you for reaching out!

This is an oversight on my part. I have not implemented a TypeHandler for std::list. I will implement this when I have the chance, in the meantime you can use a std::vector instead of the std::list. It will most likely be more performant as well.

**Reply by james5haumor** [reply]
Thank you for your response! I just wanted to express my gratitude for your tool—it's been a real lifesaver for me. Despite the issue with std::list, it's made a significant difference in my workflow. Thanks again!
